src/poprox_recommender/filters/topic.py

```diff
@@ -0,0 +1,20 @@
+from poprox_concepts import ArticleSet, InterestProfile
+
+
+class TopicFilter:
```

Comment:
I haven't yet introduced tests for the various new components, but they should be much easier to write tests for now that they're modular and decoupled from the main flow of the recommendation code